common/test/acceptance/pages/lms/login_and_register.py

```diff
@@ -63,6 +63,30 @@ def submit(self):
         return dashboard
 
 
+class ResetPasswordPage(PageObject):
+    """Initialize the page.
+
+        Arguments:
+            browser (Browser): The browser instance.
+    """
+    url = BASE_URL + "/login#forgot-password-modal"
+
+    def __init__(self, browser):
+        super(ResetPasswordPage, self).__init__(browser)
+
+    def is_browser_on_page(self):
+        return (
+            self.q(css="#login-anchor").is_present() and
+            self.q(css="#password-reset-anchor").is_present()
```

Comment:
I assume `is_present` means "in the DOM, but not necessarily visible"? In that case, i think this is good